catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/ScriptFragment.java

```diff
@@ -862,4 +874,33 @@ public void run() {
 		scriptToFocus = null;
 		brickToFocus = null;
 	}
+
+	Runnable showRunnable = new Runnable() {
+		@Override
+		public void run() {
+			fab.setVisibility(View.VISIBLE);
+		}
+	};
+
+	private void hideFab() {
+		LayoutInflater inflater2 =
+				(LayoutInflater) getActivity().getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+
+		View vi = inflater2.inflate(R.layout.activity_sprite,
+				(ViewGroup) getActivity().findViewById(R.id.activity_sprite));
+		fab = vi.findViewById(R.id.bottom_bar);
+		Handler handler = new Handler();
```

Comment:
avoid using Handler for such Ui changes. see [View.postDelayed](https://developer.android.com/reference/android/view/View#postDelayed(java.lang.Runnable,%20long))